app/uk/gov/hmrc/customs/emailfrontend/views/Layout.scala.html

```diff
@@ -81,7 +81,7 @@
                 signOutUrl = Some(uk.gov.hmrc.customs.emailfrontend.controllers.routes.SignOutController.signOut.url),
                 accessibilityStatementUrl = Some(appConfig.accessibilityLinkUrl)
             ),
-            backLink = Some(BackLink(href = backLinkUrl.getOrElse(emptyString))),
+            backLink = backLinkUrl.map(url => BackLink(href = url)),
```

Comment:
Better to refactor like below to avoid passing the empty string hence showing the backLink on the page

backLink = backLinkUrl.filter(_.nonEmpty).map(url => BackLink(href = url))